src/common/log.js

```diff
@@ -2,6 +2,7 @@
 
 const { join } = require('path')
 const { assign } = Object
```

Comment:
You can usually use spread properties everywhere in the Electron code instead of assign (everywhere but in `scripts`).